<html><head>
<META http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"><title>Campbell Biology - Sixth Edition</title><script src="../js/scripts.js" language="JavaScript"></script></head><body bgcolor="#ffffff"><table width="500" border="0" cellspacing="0" cellpadding="0"><tr><td><p class="Header2"><img src="../img/bullet.gif" align="middle" border="0">&nbsp;Genome sequences provide clues to important biological questions</p><a name="para01"></a><p class="BlockText">Genomics, the study of genomes based on their DNA sequences, is yielding new insights into fundamental questions about genome organization, the control of gene expression, growth and development, and evolution. The beauty of DNA technology is that it enables geneticists to study genes directly, without having to infer genotype from phenotype as in classical genetics. But the newer approach poses the opposite problem, determining the phenotype from the genotype. Starting with a long DNA sequence, how does one recognize genes and determine their function?</p><p class="Header3">Analyzing DNA Sequences</p><a name="para02"></a><p class="BlockText">DNA sequences are being collected in computer data banks that are available to researchers all over the world via the Internet. Scientists use software to scan the sequences for the telltale signs of protein-coding genes, such as start and stop signals for transcription and translation (promoters, for example) and sequences associated with RNA-splicing sites. The software also looks for sequences similar to those found in known genes. (Thousands of such sequences, called expressed sequence tags, or <span class="Emphasis1">ESTs,</span> are cataloged in computer databases.) In this way, researchers come up with a list of gene candidates.</p><a name="para03"></a><p class="BlockText"><span class="Emphasis2">Surprisingly Few Genes in the Human Genome.</span> The most surprising--and humbling--result to date from the Human Genome Project is the small number of putative genes. The estimated number, 30,000 to 40,000, is much lower than the 50,000 to 100,000 expected and only two to three times the number found in the fruit fly and nematode worm (TABLE 20.1). Relative to the other organisms studied so far, a much smaller fraction of human DNA is genes. Much of the enormous amount of noncoding DNA in the human genome is repetitive DNA, but unusually long introns also contribute significantly. Human introns are typically ten times longer than those of the fly or worm.</p><TABLE BORDER="0"><TR><TH BGCOLOR="#f1f1f1"><B>Table 20-1.</B>&nbsp;<span class="table_title">Genome Sizes and Numbers of Genes</span><BR></TH></TR><TR><TD><TABLE BORDER="0" CELLPADDING="5" CELLSPACING="5"><TR><TH BGCOLOR="#f1f1f1" COLSPAN="" ROWSPAN=""></TH><TH BGCOLOR="#f1f1f1" COLSPAN="" ROWSPAN=""></TH><TH BGCOLOR="#f1f1f1" COLSPAN="" ROWSPAN="">Estimated</TH><TH BGCOLOR="#f1f1f1" COLSPAN="" ROWSPAN="">Genes</TH></TR><TR><TH BGCOLOR="#f1f1f1" COLSPAN="" ROWSPAN=""></TH><TH BGCOLOR="#f1f1f1" COLSPAN="" ROWSPAN="">Genome</TH><TH BGCOLOR="#f1f1f1" COLSPAN="" ROWSPAN="">Number</TH><TH BGCOLOR="#f1f1f1" COLSPAN="" ROWSPAN="">per</TH></TR><TR><TH BGCOLOR="#f1f1f1" COLSPAN="" ROWSPAN="">Organism</TH><TH BGCOLOR="#f1f1f1" COLSPAN="" ROWSPAN="">Size</TH><TH BGCOLOR="#f1f1f1" COLSPAN="" ROWSPAN="">of Genes</TH><TH BGCOLOR="#f1f1f1" COLSPAN="" ROWSPAN="">Mb<SUP>*</SUP></TH></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"><span class="Emphasis1">H. influenzae</span></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">1.8 Mb<SUP>*</SUP></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">1,700</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">950</P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">(bacterium)</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"><span class="Emphasis1">S. cerevisiae</span></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">12 Mb</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">6,000</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">500</P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">(yeast)</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"><span class="Emphasis1">C. elegans</span></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">97 Mb</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">19,000</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">200</P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">(nematode)</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"><span class="Emphasis1">A. thaliana</span></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">100 Mb</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">25,000</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">200</P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">(plant)</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"><span class="Emphasis1">D. melanogaster</span></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">180 Mb</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">13,000</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">100</P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">(fruit fly)</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"><span class="Emphasis1">H. sapiens</span></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">3,200 Mb</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">30,000-40,000</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">10</P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">(human)</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD></TR></TABLE></TD></TR></TABLE><a name="para52"></a><p class="BlockText">So what makes humans--and vertebrate animals in general--more complex than flies or worms? For one thing, compared with those of other organisms, our genome "gets more bang for the buck" out of its coding sequences because the RNA transcripts of human genes are more subject to alternative splicing (see <span class="SmallCap">FIGURE 19.11</span>). The typical human gene probably specifies at least two or three different polypeptides by using different combinations of exons. This would bring the total number of different human polypeptides to about 90,000, without even beginning to consider the additional polypeptide diversity brought about by post-translational processing, such as cleavage or the addition of carbohydrate groups.</p><a name="para53"></a><p class="BlockText">Furthermore, the human DNA sequence (as well as other kinds of data) suggests that our polypeptides tend to be more complicated than those of invertebrates. Recall that many polypeptides consist of more than one <span class="Emphasis1">domain,</span> which is a discrete section with a particular structure and function (for some examples, see <span class="SmallCap">FIGURE 19.10</span>). Although humans do not seem to have more types of domains than invertebrates, the domains are put together in many more combinations.</p><a name="para54"></a><p class="BlockText">In summary, the human genome seems to do more mixing and matching of modular elements--exons and protein domains--than simpler organisms, and this is undoubtedly a feature of vertebrates in general.</p><a name="para55"></a><p class="BlockText"><span class="Emphasis2">Studying and Comparing Genes.</span> About half of the human genes were already known before the Human Genome Project, but what of the others? Scientists compare the sequences of new gene candidates with those of known genes from various organisms. In some cases, a newly determined gene sequence will match, at least partially, a gene whose function is well known. For example, part of a new gene may match a known gene that encodes a protein kinase, suggesting that the new gene does, too. In other cases, the new gene sequence will be similar to a sequence encountered before but of unknown function. In still other cases, the sequence may be entirely unlike anything ever seen before. In the organisms that have been sequenced so far, many of the gene candidate sequences are entirely new to science. For example, about a third of the genes of <span class="Emphasis1">E. coli,</span> the best-studied of research organisms, are new to us!</p><a name="para56"></a><p class="BlockText">Despite the high complexity of the human genome and the proteins it encodes, comparisons of genome sequences confirm very strongly the evolutionary connections between even distantly related organisms and the relevance of research on simpler organisms to our understanding of human biology. Similarities between genes of disparate organisms can be surprising, to the point that one researcher has said that he now views fruit flies as "little people with wings." Yeast, for example, has a number of genes close enough to the human versions that they can substitute for them in a human cell. In fact, researchers can sometimes work out what a human disease gene does by studying its normal counterpart in yeast. Bacterial sequences reveal unsuspected metabolic pathways that may have industrial or medical uses. Moreover, comparisons of the completed genome sequences of bacteria, archaea, and eukarya strongly support the theory that these are the three fundamental domains of life.</p><p class="Header3">Studying Gene Expression</p><a name="para57"></a><p class="BlockText">We study genomes not only to learn about individual genes and their evolution but to learn how genes act together to produce and maintain a functioning organism. A major part of the answer to how we get along with so few genes is likely to be unusually complex networks of interactions among genes and their products. While some scientists are laboring to sequence and analyze their structure, others are using the genome sequences already obtained to study patterns of gene expression from a new, global perspective. These researchers are asking which genes are transcribed in different situations. Their basic strategy is to isolate the mRNA made in particular cells, use these molecules as templates for making a cDNA library by reverse transcription, and then compare this cDNA with other collections of DNA by hybridization. Researchers can see what genes are active at different stages of development, in different tissues, or in tissues in different states of health.</p><a name="para58"></a><p class="BlockText">DNA technology makes such gene expression studies possible; automation allows them to be easily performed on a large scale. Scientists can now detect and measure the expression of thousands of genes at one time. A revolutionary new method, described in <span class="SmallCap">FIGURE 20.14</span>, uses <span class="Emphasis2">DNA microarray assays.</span> Tiny amounts of a large number of single-stranded DNA fragments representing different genes are fixed to a glass slide in a tightly spaced array (grid). (The array is also called a <span class="Emphasis1">DNA chip</span> by analogy to a computer chip.) Ideally, these fragments represent all the genes of an organism, as is already possible for organisms whose genomes have been completely sequenced. The fragments are tested for hybridization with various samples of cDNA molecules, which have been labeled with fluorescent dyes. An early test of this method compared the genes expressed in the roots and leaves of <span class="Emphasis1">Arabidopsis.</span> The researchers went on to sequence the genes that were more highly expressed in one tissue than in the other. The results were as predicted; genes encoding known photosynthetic enzymes, for instance, were turned on in the leaves but not in the roots. But more important than confirming predictions is the method’s potential for uncovering new genes and gene interactions and providing clues to gene function. For example, DNA microarray assays are being used to compare cancerous with noncancerous tissues. Studying the differences in gene expression may lead researchers to new diagnostic techniques and biochemically targeted treatments, as well as a fuller understanding of cancer. Ultimately, information from microarray assays should provide us a grander view: how ensembles of genes interact to form a living organism.</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig20-14a','../fig/big/fig20-14a.jpg','fig20-14a');"><img border="0" src="../fig/thm/fig20-14a.jpg" alt="fig20-14a" name="myfig20-14a"></a><br>a.</td></tr><tr><td valign="top" class="text"><a href="javascript:bigFig('fig20-14b','../fig/big/fig20-14b.jpg','fig20-14b');"><img border="0" src="../fig/thm/fig20-14b.jpg" alt="fig20-14b" name="myfig20-14b"></a><br>b.</td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 20-14.</B>&nbsp;<span><span class="Emphasis2">DNA microarray assay for gene expression.</span> With this method, a researcher can simultaneously test all the genes expressed in a particular tissue for hybridization with an array of short DNA sequences representing thousands of genes, even all the genes of the organism. The cDNA molecules made in part (a) step 2 and added to every well of the microarray in step 3 represent the genes that were active in the tissue. In step 4, the spots where any of the cDNA hybridizes fluoresce with an intensity indicating the relative amount of the mRNA that was in the tissue. Molecules of cDNA representing more than one tissue or the same kind of tissue under different conditions can be tested together by using a different colored label for each.</span></td></tr></table><p class="Header3">Determining Gene Function</p><a name="para59"></a><p class="BlockText">Perhaps the most interesting genes discovered in genome sequencing and gene expression studies are those whose functions are completely mysterious. How do researchers discover the functions of these genes? The main idea is to disable the gene in some way and hope that the consequences in the cell or organism will provide clues to the gene’s normal function. One powerful approach is <span class="Emphasis2"><span class="Emphasis1">in vitro</span> mutagenesis,</span> a technique that can be used to introduce specific changes into the sequence of a cloned gene. Such mutations can alter or destroy the function of the protein product. Then, when the mutated gene is returned to a cell, it may be possible to determine the function of the missing normal protein by examining the phenotype of the mutant. Researchers can even put such a mutated gene into cells from the early embryo of a multicelled organism (such as a mouse) to study the role of the gene in the development and functioning of the whole organism.</p><a name="para60"></a><p class="BlockText">Some researchers working with nonmammalian organisms have recently begun using a simpler and faster method to silence the expression of selected genes. Called <span class="Emphasis2">RNA interference (RNAi),</span> the method uses synthetic double-stranded RNA molecules matching the sequence of a particular gene to trigger breakdown of the gene’s messenger RNA. How the double-stranded RNA brings about this effect in the cell is still largely a mystery, but the process is a natural one that probably evolved to protect cells from viruses and restrain the movement of retrotransposons (which can disrupt essential genes). Scientists have only recently achieved some success in using the method to silence genes in mammalian cells, but in other kinds of organisms, such as the nematode, RNAi has already proved valuable.</p><p class="Header3">Future Directions in Genomics</p><a name="para61"></a><p class="BlockText">The success in mapping and sequencing genomes is encouraging scientists to move ahead to <span class="Emphasis2">proteomics,</span> the systematic study of the full protein sets <span class="Emphasis1">(proteomes)</span> encoded by genomes. Proteomics is a new kind of challenge. Because of alternative splicing of RNA and modifications of proteins after translation, the number of proteins in humans and our close relatives is likely to far exceed the number of genes. And collecting all the proteins will be difficult because a cell’s proteins differ with the cell type and its state. Moreover, unlike DNA, proteins are extremely varied in structure and chemical and physical properties. However, because proteins are the molecules that actually carry out the activities of the cell, we must study them to learn how cells and organisms function. Ongoing technical advances, such as the recent development of protein micro-arrays for studying protein interactions, will be necessary to meet the challenge.</p><a name="para62"></a><p class="BlockText">Genomics and proteomics are giving biologists an increasingly global perspective on the study of life. In an essay in <span class="Emphasis1">Science,</span> noted MIT biologists Eric Lander and Robert Weinberg predict that the availability of "complete parts lists of organisms"--that is, catalogs of genes and proteins--will change the discipline of biology dramatically:</p><a name="para63"></a><p class="ExtractText">The 21st century discipline will focus increasingly on the study of entire biological systems, by attempting to understand how component parts collaborate to create a whole. For the first time in a century, reductionists [are yielding] ground to those trying to gain a holistic view of cells and tissues.<A href="javascript:popNote('20','1')"><SUP><B>*</B></SUP></A></p><a name="para64"></a><p class="BlockText">Advances in <span class="Emphasis2">bioinformatics,</span> the application of computer science and mathematics to genetic and other biological information, will play a crucial role in dealing with the enormous masses of data involved.</p><a name="para65"></a><p class="BlockText">As Landers and Weinberg point out elsewhere in their essay, we are now poised to understand the spectrum of genetic variation in humans. Because the history of the human species is so short--we are probably all descended from a small population living in Africa 150,000 to 200,000 years ago--the amount of DNA variation in humans is small compared to that of other species. Most of our diversity seems to be in the form of <span class="Emphasis2">single nucleotide polymorphisms</span> (<span class="Emphasis2">SNPs,</span> pronounced "snips"), which are single base-pair variations in the genome.<A href="javascript:popNote('20','2')"><SUP><B>*</B></SUP></A></p><a name="para66"></a><p class="BlockText">In the human genome, SNPs occur on average about once in 1,000 base pairs. In other words, if you could compare your personal DNA sequence with that of the person sitting next to you--or with that of someone on the other side of the world--you would find them to be 99.9%  identical.</p><a name="para67"></a><p class="BlockText">Scientists are already well on their way to identifying the locations of the 3 million or so SNP sites in the human genome. These will be useful genetic markers for studying human evolution, the differences between human populations, and the migratory routes that led from Africa to the other continents. They will also be valuable markers for identifying disease genes and genes that affect our health in more subtle ways by influencing susceptibility to certain diseases or sensitivities to environmental toxins or drugs. This is likely to change the practice of medicine later in the 21st century. However, applications of DNA research and technology are already affecting our lives in many ways. We devote the rest of the Chapter to a discussion of some of these applications and related issues.</p></td></tr></table></body></html>
